pyxrf/gui.py

```diff
@@ -69,16 +78,10 @@ def run():
     # Output log to gui, turn off for now
     # error at mac, works fine on linux
     # so log info only outputs to terminal for now.
-    formatter = logging.Formatter(fmt='%(asctime)s : %(levelname)s : %(message)s')
     guihandler = GuiHandler()
     guihandler.setLevel(logging.INFO)
     guihandler.setFormatter(formatter)
-    stream_handler = logging.StreamHandler()
-    stream_handler.setFormatter(formatter)
-    stream_handler.setLevel(logging.INFO)
-    logger.setLevel(logging.INFO)
     logger.addHandler(guihandler)
-    logger.addHandler(stream_handler)
```

Comment:
+1 for separating those 2 handlers.

Reply:
The handler was added after GUI initialization, so messages was printed in the terminal.